app/models/patient.rb

```diff
@@ -1,6 +1,6 @@
 class Patient
-  attr_accessor :fname, :lname, :pid, :fhir
-  attr_writer :observations, :conditions, :medications
+  attr_accessor :fname, :lname, :pid, :fhir, :observations
+  attr_writer :conditions, :medications
```

Comment:
Changing `observations` from an `attr_writer` to an `attr_accessor` has no effect, since the `observations` method will overwrite the reader created by `attr_accessor`. I'm going to change this back to make its intent a little clearer.